provider/provider_yaml_test.go

```diff
@@ -201,7 +201,9 @@ func TestAutoExtractedProgramsUpgrade(t *testing.T) {
 		{"cloudrunv2-job-6"},
 		{"cloudrunv2-service-4"},
 		{"cloudrunv2-service-5"},
-		{"cloudrunv2-service-7"},
+		// TODO: This upgrade will no longer be possible, as it touches multiple new changes. Must be re-recorded.
+		// Skipping for now.
+		//{"cloudrunv2-service-7"},
```

Comment:
This needs to be addressed or reference an a associated issue before merging.

Reply:
Issue filed and linked.